modules/oxide/gulpfile.js

```diff
@@ -90,7 +92,14 @@ gulp.task('minifyCss', function() {
   return gulp.src(['./build/skins/**/*.css', '!**/*.min.css'])
     .pipe(sourcemaps.init())
     .pipe(cleanCSS({ rebase: false }))
-    .pipe(rename({ extname: '.min.css' }))
+    .pipe(through2.obj(function(file, _, cb) {
+      if (file.isBuffer()) {
+        const contents = `tinymce.resources.add('${file.relative}', '${file.contents.toString().replace(/'/g, "\\'")}')`;
```

Comment:
I think it's better to use JSON.stringify here so make sure that all things gets escaped properly.

Reply:
Which part do you want to put stringify on? The file is css, so it shouldn't have anything to escape in the first place?

Reply:
CSS has plenty of things that might need escaping.

All this is valid css:
```
[foo="bar"], [foo='bar'] {}
.foo { background: url('bar') };
.foo { background: url("bar") };
```

Reply:
Might be other characters in there as well. Just replacing the quotes just seems a bit weak that all. JSON.stringify will make sure that it's always a valid string.

Reply:
Alright. Can do that.